zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/slack/SlackNotifier.java

```diff
@@ -26,10 +42,116 @@ public void collectionAlarm(Collection c, String alarm, PostMessageField... args
      * Send a collection specific alarm to Slack.
      *
      * @param alarm - the string message to apply to the notification.
-     * @param args - additional arguments to add to the notification.
+     * @param args  - additional arguments to add to the notification.
      */
     @Override
     public void alarm(String alarm, PostMessageField... args) {
         SlackNotification.alarm(alarm, args);
     }
+
+
+    @Override
+    public void sendSlackMessage(PostMessage message) throws Exception {
+        try {
+            slackClient.sendMessage(message);
+        } catch (Exception ex) {
+            throw new Exception("unexpected error sending slack message", ex);
+        }
+    }
+
+    @Override
+    public PostMessage createPostMessage(String channel, String text) {
+        return profile.newPostMessage(channel, text);
+    }
+
+    @Override
+    public boolean sendCollectionAlarm(Collection collection, String channel, String customMessage, Exception ex) {
+        boolean result = validate(collection, channel, customMessage, "alarm");
+        if (!result) {
+            return false;
+        }
```

Comment:
```
if !validate(collection, channel, customMessage, "alarm") {
    return false;
}
```
Functionally the same just shorter 😄 . Generally speaking I would only assign the result to a variable if I need to access it multiple times. It's also good to be consistent in how you return from a method:
Either do:
```
if !validate(collection, channel, customMessage, "alarm") {
    return false;
}
```
or
```
boolean isValid = validate(collection, channel, customMessage, "alarm");
if (!isValid) {
    return false;
}
```
But don't mix and match. I know it seems like a small point but you should try either follow the existing style thats in place or choose your own approach and implement it consistently - this helps to keep the code readable.

I am being really picky with this comment, the rest of this code is looking good so if you address this comment we can merge this in.